src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,4 +1,5 @@
-// Add the required props
-export const UserInfo = () => (
-  <>UserInfo markup</>
+export const UserInfo = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user.email}`}>
```

Comment:
You should add a check to make sure the prop 'user' exists before trying to display the user's email and name. This will prevent the application from crashing if 'user' is undefined.